ccd-claim-migration/src/main/java/uk/gov/hmcts/cmc/ccd/migration/config/EtlDbConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.hmcts.cmc.ccd.migration.config;
+
+import org.skife.jdbi.v2.DBI;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.jdbc.datasource.TransactionAwareDataSourceProxy;
+import org.springframework.transaction.PlatformTransactionManager;
+import uk.gov.hmcts.cmc.ccd.migration.repositories.ClaimRepository;
+
+import javax.sql.DataSource;
+
+@Configuration
+public class EtlDbConfiguration {
+
+    @Bean
+    @Primary
+    public DataSource getDataSource() {
+        return DataSourceBuilder
+            .create()
+            .url("jdbc:postgresql://localhost:5430/")
```

Comment:
shouldn't be using the properties in `application.yml` rather than having hardcoded ehre?

Reply:
yes